src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/jpa/entity/Cas1OutOfServiceBedRevisionEntity.kt

```diff
@@ -0,0 +1,75 @@
+package uk.gov.justice.digital.hmpps.approvedpremisesapi.jpa.entity
```

Comment:
thanks for the excellent explanation here. That will really help maintenance, I believe. @

Just to note that the commit title should maybe change to from "... out-of-service bed details" to "... out-of-service bed revisions"?